backend/src/impl/benchmark_configs/config_NER.json

```diff
@@ -0,0 +1,106 @@
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.tam", "splits": {"train": 15000, "validation": 1000, "test": 1000}, "languages": ["tam"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.tel", "splits": {"train": 1000, "validation": 1000, "test": 1000}, "languages": ["tel"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.tha", "splits": {"train": 20000, "validation": 10000, "test": 10000}, "languages": ["tha"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.tgl", "splits": {"train": 10000, "validation": 1000, "test": 1000}, "languages": ["tgl"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.tur", "splits": {"train": 20000, "validation": 10000, "test": 10000}, "languages": ["tur"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.urd", "splits": {"train": 20000, "validation": 1000, "test": 1000}, "languages": ["urd"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.vie", "splits": {"train": 20000, "validation": 10000, "test": 10000}, "languages": ["vie"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.yor", "splits": {"train": 100, "validation": 100, "test": 100}, "languages": ["yor"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]},
+{"dataset_name": "xtreme", "dataset_class_name": "Xtreme", "sub_dataset": "PAN-X.cmn", "splits": {"train": 20000, "validation": 10000, "test": 10000}, "languages": ["cmn"], "task_categories": [["sequence-labeling"]], "tasks": ["named-entity-recognition"]}
+
+    ],
+    "views": [
+        {
+            "name": "Linguistic-weighted Global Average",
+            "operations": [
+                {"op": "max", "group_by": "languages", "skip_groupby_system": true},
+                {"op": "mean", "skip_groupby_system": true}
+            ]
+        },
+        {
+            "name": "Demographic-weighted Global Average",
+            "operations": [
+                {"op": "max", "group_by": "languages", "skip_groupby_system": true},
+                {"op": "weighted_sum", "weight": 1, "skip_groupby_system": true}
```

Comment:
The `pop_weight` part still needs to be implemented like the `masakhaner` benchmark I guess.

Reply:
Sure, so I guess we will be keeping a dictionary of all relevant languages and their population weights; will have to be done pretty manually and be updated as necessary. Do you have any suggestions on where I should put this? I was thinking just in benchmark_utils.py ...